decidim-system/app/commands/decidim/system/create_organization.rb

```diff
@@ -33,8 +33,8 @@ def call
           CreateDefaultContentBlocks.call(@organization)
           invite_form = invite_user_form(@organization)
           invitation_failed = invite_form.invalid?
+          raise ActiveRecord::RecordInvalid if invite_form.invalid?
```

Comment:
There are a couple problems with this approach:

1. After this change, we've a variable (`invitation_failed`) that's no longer used, so we should remove it (lines 27 and 35)
2. With the current approach, we're raising `RecordInvalid` and then rescuing it with a generic invalid error. We should give meaningful error messages, to help the user actually fix the form. This change will generate changes in the controller to present the error itself that was broadcasted. 

This is how it should look like with these changes:

```diff
diff --git a/decidim-system/app/commands/decidim/system/create_organization.rb b/decidim-system/app/commands/decidim/system/create_organization.rb
index 1f734a4e02..c6f4d7860e 100644
--- a/decidim-system/app/commands/decidim/system/create_organization.rb
+++ b/decidim-system/app/commands/decidim/system/create_organization.rb
@@ -2,6 +2,9 @@

 module Decidim
   module System
+    class InvitationFailedError < ActiveRecord::RecordInvalid
+    end
+
     # A command with all the business logic when creating a new organization in
     # the system. It creates the organization and invites the admin to the
     # system.
@@ -24,7 +27,6 @@ module Decidim

         @organization = nil
         invite_form = nil
-        invitation_failed = false

         transaction do
           @organization = create_organization
@@ -32,13 +34,14 @@ module Decidim
           CreateDefaultHelpPages.call(@organization)
           CreateDefaultContentBlocks.call(@organization)
           invite_form = invite_user_form(@organization)
-          invitation_failed = invite_form.invalid?
-          raise ActiveRecord::RecordInvalid if invite_form.invalid?
+          raise InvitationFailedError if invite_form.invalid?
         end

         Decidim::InviteUser.call(invite_form) if @organization && invite_form

         broadcast(:ok)
+      rescue InvitationFailedError
+        broadcast(:invalid_invitation)
       rescue ActiveRecord::RecordInvalid, ActiveRecord::RecordNotUnique
         broadcast(:invalid)
       end
diff --git a/decidim-system/app/controllers/decidim/system/organizations_controller.rb b/decidim-system/app/controllers/decidim/system/organizations_controller.rb
index f3236873b5..18da2d798b 100644
--- a/decidim-system/app/controllers/decidim/system/organizations_controller.rb
+++ b/decidim-system/app/controllers/decidim/system/organizations_controller.rb
@@ -22,6 +22,11 @@ module Decidim
             redirect_to organizations_path
           end

+          on(:invalid_invitation) do
+            flash.now[:alert] = t("organizations.create.error_invitation", scope: "decidim.system")
+            render :new
+          end
+
           on(:invalid) do
             flash.now[:alert] = t("organizations.create.error", scope: "decidim.system")
             render :new
diff --git a/decidim-system/config/locales/en.yml b/decidim-system/config/locales/en.yml
index 944f484279..f030e881c5 100644
--- a/decidim-system/config/locales/en.yml
+++ b/decidim-system/config/locales/en.yml
@@ -143,6 +143,7 @@ en:
           show: Show advanced settings
         create:
           error: There was a problem creating a new organization.
+          error_invitation: There was a problem creating a new organization. Review your organization admin name.
           success_html: |
             <p>
               Organization successfully created.
```